common/types/time.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "time"
+
+const (
+	TimeFormat = "2006-January-02 15:04:05"
+)
+
+func ConvertUnixTimestampToString(val uint64) string {
+	return time.Unix(int64(val), 0).Format(TimeFormat)
```

Comment:
what hapens if you just time.Unix(int64(val), 0).string() ?

Reply:
It will use the default format. I chose not to use the default format since in Hebrew format, the month come first, and in other languages, the month comes second after the day. I wanted it to be clear what is the month and what is the day, and that is why I've created the TimeFormat for.